{{cookiecutter.project_slug}}/clients/web/vue3/src/services/users/api.js

```diff
@@ -18,18 +23,22 @@ const requestPasswordReset = createCustomServiceCall(
   {
     inputShape: forgotPasswordShape,
   },
-  async ({ client, input }) => {
-    await client.get(`/password/reset/${input.email}/`)
+  async ({ client, input, utils }) => {
+    const res = await client.post('/password/reset/', utils.toApi(input))
+    return utils.fromApi(res.data)
   },
 )
+
 const resetPassword = createCustomServiceCall(
   {
-    inputShape: { uid: z.string().email(), token: z.string(), password: z.string() },
+    inputShape: resetPasswordShape,
     outputShape: userShape,
   },
   async ({ client, input, utils }) => {
-    const { email, ...rest } = utils.toApi(input)
-    const res = await client.post(`/password/reset/code/confirm/${input.email}/`, rest)
+    const { password } = utils.toApi(input)
+    const res = await client.post(`/password/reset/confirm/${input.uid}/${input.token}/`, {
+      password,
+    })
```

Comment:
This was fixed a while ago but somehow got re-broken in the past couple of months.